sqlmesh/core/engine_adapter/base_postgres.py

```diff
@@ -24,6 +24,7 @@ class BasePostgresEngineAdapter(EngineAdapter):
     DEFAULT_BATCH_SIZE = 400
     COMMENT_CREATION_TABLE = CommentCreationTable.COMMENT_COMMAND_ONLY
     COMMENT_CREATION_VIEW = CommentCreationView.COMMENT_COMMAND_ONLY
+    SUPPORTED_DROP_CASCADE_OBJECT_KINDS = ["SCHEMA", "TABLE", "VIEW"]
```

Comment:
IMHO I don't know if we should ever apply this for tables. I realized that this was added for completeness sake, but I'd rather make a conscious choice for supporting this when the need arises.